### ISSUE TITLE
Follow Up: write REQUEST type iMIP messages to calendars automatically

### ISSUE THREAD

**Issue by miaulalala** [nextcloud/mail#6997]
### Is your feature request related to a problem? Please describe.

This is a follow up to https://github.com/nextcloud/mail/issues/6802

The difficulty is which calendar the invitation should be written to. 

Options:
1) User can define this invitation calendar as a default option somewhere
2) always write to the personal calendar and let the user handle the moving of calendar objects in the Calendar app

### To Do

- [ ]  Add an if clause in the IMIpService::process function that checks for 'REQUEST'. This clause should then 
    - [ ] either check for the existence of the personal calendar or
    - [ ] check for the preset calendar from oc_prefrences

- [ ] process the REQUEST by calling the ICreateFromString::createFromString method and writing to that specific calendar. 
- [ ] Mark the message as processed, or in case the storing didn't work, mark it as erroneous.

- [ ] In the frontend, if we go with option 1), add a settings field somewhere and store it in oc_preferences.

### Describe alternatives you've considered

Not processing the messages and leaving it up to the user if they want to see them in their calendar. We already provide an import option.

**Comment by miaulalala** [comment]
As far as I can tell, the invitations are already written to the default calendar. @st3iny can you confirm?

**Comment by ChristophWurst** [comment]
@st3iny can you?

**Comment by st3iny** [comment]
Yes, I can. We have a configurable default calendar for incoming invitations which we can simply reuse here.

**Comment by st3iny** [comment]
I think the biggest concern here is how to avoid spam? We don't want to flood a user's calendar with unsolicited invitations.
